src/main/java/org/birthdaynotifier/service/impl/DayServiceImpl.java

```diff
@@ -14,59 +14,54 @@
 @Service
 public class DayServiceImpl implements DayService {
 
-  private final DayRepository dayRepository;
+    private final DayRepository dayRepository;
 
-  private final DayMapper dayMapper;
+    private final DayMapper dayMapper;
 
-  public DayServiceImpl(DayRepository dayRepository, DayMapper dayMapper) {
-    this.dayRepository = dayRepository;
-    this.dayMapper = dayMapper;
-  }
-
-  @Override
-  public List<DayDto> getAll() {
-    Optional<List<Day>> optionalDays = dayRepository.findAll();
-    List<Day> dayList = optionalDays.orElseThrow(() -> new RuntimeException("days not found"));
-    return dayMapper.toDto(dayList);
-  }
+    public DayServiceImpl(DayRepository dayRepository, DayMapper dayMapper) {
+        this.dayRepository = dayRepository;
+        this.dayMapper = dayMapper;
+    }
 
-  @Override
-  public DayDto getById(Long id) {
-    Optional<Day> optionalDay = dayRepository.findById(id);
-    Day day = optionalDay.orElseThrow(() -> new RuntimeException("day not found"));
-    return dayMapper.toDto(day);
-  }
+    @Override
+    public List<DayDto> getAll() {
+        Optional<List<Day>> optionalDays = dayRepository.findAll();
+        List<Day> dayList = optionalDays.orElseThrow(() -> new RuntimeException("days not found"));
+        return dayMapper.toDto(dayList);
```

Comment:
Так должно работать и выглядит лучше
```suggestion
        return dayRepository
            .findAll()
            .map(dayMapper::toDto)
            .orElseThrow(() -> new RuntimeException("days not found"));
```

Reply:
По аналогии можно сделать в других местах